package.json

```diff
@@ -243,15 +245,19 @@
     "karma-qunit": "^1.2.1",
     "koa": "^2.7.0",
     "lockfile-lint": "^4.0.0",
+    "merge-deep": "^3.0.2",
```

Comment:
btw while including lots of minimal `lodash.*` packages sounds like a good idea, i found they tend to have a lot of overhead and redundant code, so its actually smaller to take the whole of lodash

Reply:
Yep; we brought in the entirely of `lodash` for that reason. Also because the entirely of `lodash` was already present 10+ times as a transitive dependency.